clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/deploy/DescriptionAuthorizer.java

```diff
@@ -60,6 +61,7 @@ public DescriptionAuthorizer(
     this.fiatPermissionEvaluator = fiatPermissionEvaluator.orElse(null);
     this.opsSecurityConfigProps = opsSecurityConfigProps;
 
+    this.skipAuthorizationId = registry.createId("authorization.skip");
```

Comment:
Not a very descriptive metric name - skip what?

Reply:
Hm, I can change this to the past tense `authorization.skipped` - that seems more descriptive to me (basically, this metric is saying that we skipped authorization on a resource).

Reply:
That sounds good.